SUMMARY.md

```diff
@@ -41,6 +40,52 @@
    * [Waterline](databases/waterline.md)
    * [LevelUP](databases/levelup.md)
    * [RethinkDB](databases/rethinkdb.md)
+* [Ecosystem & APIs](/ecosystem/readme.md)
+   * [authentication-popups](/ecosystem/authentication-popups/readme.md)
```

Comment:
I'm thinking in that case it'd become its own section anyway (not under Ecosystem).